# Enable state [true/false]
bool state
---
# Response value
int64 response

